app/pages/SiloAccessPage.tsx

```diff
@@ -162,6 +164,12 @@ export function SiloAccessPage() {
     <>
       <PageHeader>
         <PageTitle icon={<Access24Icon />}>Access &amp; IAM</PageTitle>
+        <DocsPopover
+          heading="Access"
+          icon={<Access16Icon />}
+          summary=""
```

Comment:
I didn't have a great summary for this; I think "Access" is pretty straightforward without a description, though.

Reply:
nvm

Reply:
> Configure user and group access to this <project/silo>.

🤷

Reply:
I like yours better!